ballista/ui/scheduler/src/components/ExecutorsList.tsx

```diff
@@ -56,16 +61,12 @@ const columns: Column<any>[] = [
   },
 ];
 
-interface NodesListProps {
-  nodes: NodeInfo[];
-}
-
-export const NodesList: React.FunctionComponent<NodesListProps> = ({
-  nodes = [],
+export const ExecutorsList: React.FunctionComponent<ExecutorsListProps> = ({
+  executors = [],
 }) => {
   return (
     <Box flex={1}>
-      <DataTable maxW={960} columns={columns} data={nodes} pageSize={4} />
+      <DataTable maxW={960} columns={columns} data={executors} pageSize={4} />
```

Comment:
The `maxW` property is not really useful, unless we need this width size restriction. Removing it makes the table look better.